### PR TITLE
Merchant order show page#7

### LINKED ISSUE #7: User Story 59, Merchant sees an order show page
```
As a merchant
When I visit an order show page from my dashboard
I see the customer's name and address
I only the items in the order that are being purchased from my inventory
I do not see any items in the order being purchased from other merchants
For each item, I see the following information:
- the name of the item, which is a link to my item's show page
- a small thumbnail of the item
- my price for the item
- the quantity the user wants to purchase
```

### Checklist:

- [ ] base code tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #8

depends on #30

### PR DESCRIPTION
**Pull request by asmolentzov**
Completes User Story 59, Merchant sees an order show page
- Adds content to the merchant order show page: 
  - Customer information (name and address)
  - Item information (name, thumbnail, price, quantity the customer wants to purchase)
- Adds Order instance method to return the specified merchant's items in that order, along with their quantity
- Adds controller logic 
- Adds controller logic so only merchants can access the page
- All tests passing, 99.96% coverage
Closes #7

### CODE REVIEW

File: app/models/order.rb [hunk 1]
```diff
@@ -44,4 +44,11 @@ def merchant_items_value(merchant_id)
     .where("items.user_id = ?", merchant_id)
     .sum("order_items.order_price")
   end
+  
+  def merchant_items_with_quantity(merchant_id)
+    items = self.items.where("items.user_id = ?", merchant_id)
+    items.map do |item|
+      [item, self.order_items.find_by(item_id: item.id).quantity]
+    end
```
Comment (Bradniedt) [inline app/models/order.rb]: Nice!

File: app/views/orders/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  </p>
+</div>
+<% @items_with_quantity.each do |item, quantity| %>
+  <div id="item-<%= item.id %>">
+    <%= image_tag(item.image_link, class:"thumbnails") %>
+    <ul>
+      <li><%= link_to item.name, item_path(item) %></li>
+      <li>Price: <%= number_to_currency(item.current_price / 100) %></li>
+      <li>Quantity: <%= quantity %></li>
+    </ul>
```
Comment (Bradniedt) [inline app/views/orders/show.html.erb]: Well done! I like the addition of the tags/labels for each piece like "Price:"
